api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java

```diff
@@ -315,9 +319,11 @@ public static Short organizationTypeToStorage(OrganizationType organizationType)
               .put(PrePackagedConceptSetEnum.FITBIT_HEART_RATE_LEVEL, (short) 5)
               .put(PrePackagedConceptSetEnum.FITBIT_HEART_RATE_SUMMARY, (short) 6)
               .put(PrePackagedConceptSetEnum.FITBIT_INTRADAY_STEPS, (short) 7)
-              .put(PrePackagedConceptSetEnum.FITBIT, (short) 8)
-              .put(PrePackagedConceptSetEnum.WHOLE_GENOME, (short) 9)
-              .put(PrePackagedConceptSetEnum.ZIP_CODE_SOCIOECONOMIC, (short) 10)
+              .put(PrePackagedConceptSetEnum.FITBIT_SLEEP_DAILY_SUMMARY, (short) 8)
+              .put(PrePackagedConceptSetEnum.FITBIT_SLEEP_LEVEL, (short) 9)
```

Comment:
This will introduce a bug, as you are rewriting the enum persistent values. This means that any existing dataset will load anything persisted with 8 or 9 as FITBIT_SLEEP_DAILY_SUMMARY or FITBIT_SLEEP_LEVEL instead of FITBIT or WHOLE_GENOME. New enum values always have to be injected at the end of the BiMap.
Example: 
.put(PrePackagedConceptSetEnum.FITBIT_SLEEP_DAILY_SUMMARY, (short) 108)
.put(PrePackagedConceptSetEnum.FITBIT_SLEEP_LEVEL, (short) 11)

Reply:
done. Updated order in workbench-api.yaml to match the predefined DbStorageEnum for PrePackaged ConceptSet and Domain and added tests to ensure manual values in DbStorageEnum is preserved.